java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -849,6 +850,118 @@ public static CompletableFuture<Long> forget(
         return executeCommand(client, new GlideString[] {gs(JSON_FORGET), key, path});
     }
 
+    /**
+     * Toggles a Boolean value stored at the root within the JSON document stored at <code>key</code>.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @return Returns an <code>Object[]</code> with a list of integers for the root, with the toggled
+     *     boolean value, or <code>null</code> for JSON values matching the root that are not boolean.
+     *     If <code>key</code> doesn't exist, returns <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", ".", true).get();
+     * var res = Json.toggle(client, "doc").get();
+     * assert res.equals(false);
```

Comment:
your example returns a boolean.  Your method signature returns an Object.  The @return in the javadoc says is returns a list of integers.  
Which is correct?

Reply:
I think you need to update the example.  
And the return value should be Boolean[].  
You may need to update value_conversion and add JSON.TOGGLE to: https://github.com/valkey-io/valkey-glide/blob/release-1.2/glide-core/src/client/value_conversion.rs#L1411

Reply:
toggle without path returns a single bool (not in array)

Reply:
So you can simplify examples
```suggestion
     * assert !res;
```

Reply:
The example is correct. I have verified on EC2.
I will update the documentation.